### PR TITLE
PP-5185 - settings page redundant for direct debit gateways

### PR DESCRIPTION
**Pull request by jonheslop**
Settings page errored because it couldnt retrieve card only settings on a DD gateway, so redirect from this page to API Keys, which is basically what used to happen when you clicked settings

### CODE REVIEW

File: test/cypress/integration/settings/settings-index.spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        {
+          name: 'getDirectDebitGatewayAccountSuccess',
+          opts: {
+            gateway_account_id: gatewayAccountId
+          }
+        }
+      ])
+    })
+
+    it('should show it is disabled', () => {
```
Comment (stephencdaly) [inline test/cypress/integration/settings/settings-index.spec.js]: what is disabled?

File: test/cypress/integration/settings/settings-index.spec.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      cy.setEncryptedCookies(userExternalId, gatewayAccountId)
+      cy.request({
+        url: '/settings/',
+        followRedirect: false // turn off following redirects
+      })
+        .then((resp) => {
+          // redirect status code is 302
+          expect(resp.status).to.eq(302)
+          expect(resp.redirectedToUrl).to.contain('/api-keys')
+        })
```
Comment (stephencdaly) [inline test/cypress/integration/settings/settings-index.spec.js]: could you do `cy.visit('/settings'`) then check the location? would that make it more of a cypress-style test?

File: test/cypress/integration/settings/settings-index.spec.js (hunk 1 above)
Comment (jonheslop) [reply test/cypress/integration/settings/settings-index.spec.js]: Oh lol left over from test I copied will update words

File: test/cypress/integration/settings/settings-index.spec.js (hunk 2 above)
Comment (jonheslop) [reply test/cypress/integration/settings/settings-index.spec.js]: The problem then is it redirects to `/api-keys` which calls publicApi and there are no fixtures for that yet, so this was quicker 😬

Comment (stephencdaly) [reply test/cypress/integration/settings/settings-index.spec.js]: ahh ok, that's fine then

**Comment by jonheslop** [comment]
Closing in favour of a much simpler solution! https://github.com/alphagov/pay-selfservice/pull/1516

**Pull request closed without merging**